src/components/PageSizeSelector.tsx

```diff
@@ -38,20 +41,38 @@ export const PageSizeSelector: React.FC<Props> = ({
       sendAnalyticsEvent();
     });
 
-  return (
+  return useLeafygreen ? (
     <Select
+      aria-labelledby="page-size-select"
+      size="small"
+      value={value.toString()}
+      onChange={(pageSize: string) => handleChange(parseInt(pageSize, 10))}
+      allowDeselect={false}
+      css={css`
+        width: 130px;
+      `}
```

Comment:
Is there any reason not to use the `styled(Select)` pattern here? If not that's preferable!